openmrs/src/main/java/org/motechproject/openmrs/tasks/OpenMRSTaskDataProvider.java

```diff
@@ -220,22 +219,21 @@ private ProgramEnrollmentListResult getProgramEnrollments(String lookupName, Map
             }
         }
 
-        ProgramEnrollmentListResult filteredProgramEnrollments = prepareProgramEnrollmentListResult(programEnrollments, lookupFields, lookupFields.get(PROGRAM_NAME), lookupFields.get(ACTIVE_PROGRAM));
+        ProgramEnrollmentListResult filteredProgramEnrollments = prepareProgramEnrollmentListResult(programEnrollments, lookupFields);
 
         return filteredProgramEnrollments;
     }
 
-    private ProgramEnrollmentListResult prepareProgramEnrollmentListResult(List<ProgramEnrollment> programEnrollments, Map<String, String> lookupFields, String programName, String activeProgram) {
+    private ProgramEnrollmentListResult prepareProgramEnrollmentListResult(List<ProgramEnrollment> programEnrollments, Map<String, String> lookupFields) {
         ProgramEnrollmentListResult result = new ProgramEnrollmentListResult();
-        String programEnrollmentUuid = lookupFields.get(PROGRAM_ENROLLMENT_UUID);
-        String personUuid = lookupFields.get(PERSON_UUID);
+        String programName = lookupFields.get(PROGRAM_NAME);
+        String patientUuid = lookupFields.get(PATIENT_UUID);
```

Comment:
If user choose lookup with 'MOTECH ID' field, then 'PATIENT_UUID' will be empty.